### DISCUSSION TITLE
TLS connection for go quorum nodes

### DISCUSSION THREAD

**Discussion by trinayanbhatt** [ConsenSys/quorum#1637] [Q&A]
Hi Team,

Can someone please let us know if there is a way to enable TLS connection between goQuorum nodes(accessing it over https)?

For Private transaction manager we have the option to enable it but we can't find it for Quorum nodes.

**Comment by antonydenyer** [comment]
No, TLS is not supported on the devp2p layer. If you want your network to be completely private you should revert to normal network topology strategies to prevent data leaking.

**Comment by trinayanbhatt** [comment]
Thanks for your response. 

Can you please explain what sort of network topology will be useful here to make it more secure?
